### PR TITLE
Added Social media login.

### PR DESCRIPTION
**Pull request by Tanabodee-Yambangyang**

### CODE REVIEW

**Review by Supakrit65 — APPROVED** [review]
**look nice**

File: pdtx/urls.py [hunk 1]
```diff
@@ -15,10 +15,13 @@
 """
 from django.contrib import admin
 from django.urls import include, path
+from django.views.generic import TemplateView
 
 urlpatterns = [
     path("__reload__/", include("django_browser_reload.urls")),
     path("admin/", admin.site.urls),
     path("accounts/", include("django.contrib.auth.urls")),
```
Comment (GToidZ) [inline pdtx/urls.py:23]: Delete this line.
```suggestion

```

File: pdtx/urls.py [hunk 2]
```diff
@@ -15,10 +15,13 @@
...
     path("", include("wongnung.urls")),
+    path('', TemplateView.as_view(template_name="registration/login.html")),
```
Comment (GToidZ) [inline pdtx/urls.py:25]: Also, delete this line.
```suggestion

```

**Review by GToidZ — CHANGES_REQUESTED** [review]
There are duplicate entries in `urls.py` please fix or apply suggestions before merging.

File: pdtx/urls.py [hunk 3]
```diff
@@ -15,10 +15,13 @@
 """
 from django.contrib import admin
 from django.urls import include, path
+from django.views.generic import TemplateView
```
Comment (GToidZ) [inline pdtx/urls.py:18]: This import is not required.
```suggestion

```

**Review by GToidZ — APPROVED** [review]

**Review by Savetang19 — APPROVED** [review]
It's OK to merge!!!

**Comment by GToidZ** [comment]
[Backend: OAuth](https://trello.com/c/XxptFsum/51-backend-oauth)

**Pull request merged**